extensions/hazelcast-3-connector/hazelcast-3-connector-common/src/main/java/com/hazelcast/connector/Hz3Sinks.java

```diff
@@ -16,21 +16,55 @@
 
 package com.hazelcast.connector;
 
+import com.hazelcast.client.config.ClientConfig;
 import com.hazelcast.function.FunctionEx;
 import com.hazelcast.jet.core.Processor;
 import com.hazelcast.jet.core.ProcessorMetaSupplier;
 import com.hazelcast.jet.impl.pipeline.SinkImpl;
 import com.hazelcast.jet.pipeline.Sink;
 import com.hazelcast.jet.pipeline.Sinks;
+import com.hazelcast.spi.annotation.Beta;
+import com.hazelcast.spi.properties.ClusterProperty;
 
 import javax.annotation.Nonnull;
 import java.util.Map;
 
 import static com.hazelcast.jet.core.ProcessorMetaSupplier.preferLocalParallelismOne;
 
 /**
- * Sinks to connect to Hazelcast 3 cluster
+ * Provides sinks to write to Hazelcast 3 cluster.
+ * <p>
+ * The usage is similar to {@link Sinks#remoteMap(String, ClientConfig)}.
+ * Because of incompatible APIs the configuration is passed as an XML
+ * document in a string - note that the XML configuration must conform
+ * to 3.x schema.
```

Comment:
Here, maybe we can direct the user to: https://www.hazelcast.com/schema/config/. It's entirely up to you, mine is just as a optional suggestion (if added here it should be added to Hz3Sources as well).